### PR TITLE
Automated generate of overview cards

### PR DESCRIPTION
**Pull request by weijunyu**
Overview sections now do not need to be hard-coded; the list of cards would render with each document's `title` and `description` fields.

### CODE REVIEW

File: collections/_technologies/digital-solutions-to-address-covid-19/tracetogether.html [hunk 1]
```diff
@@ -24,13 +27,16 @@ <h2>TraceTogether - Community-driven Contact Tracing</h2>
...
+  You may access the TraceTogether Token Technical Write-up
+  <a
+    href="/assets/files/TT_Token_Technical_Writeup.pdf"
+    rel="noopener noreferrer nofollow"
+    >here</a
+  >.
+</p>
 
-<hr>
+<hr />
```
Comment (teochengyong) [inline collections/_technologies/digital-solutions-to-address-covid-19/tracetogether.html:39]:
```suggestion
<hr>
```

**Review by teochengyong — COMMENTED** [review]
Some questions on commented content =)

Always happy to see that we remove codes!

File: apps/src/_cards.scss [hunk 2]
```diff
@@ -21,6 +21,15 @@
   background-color: #f7f7f7;
 }
 
+// .sgds-card-button {
```
[the remark this answers is not in the document]
Comment (weijunyu) [reply apps/src/_cards.scss:24]: Removing this css as it's unused; was experimenting earlier with making the whole card a link.

File: apps/src/terms/TermsApp.vue [hunk 3]
```diff
@@ -18,7 +18,7 @@
         </div>
       </div>
     </div>
-    <div class="row">
+    <!-- <div class="row">
```
[the remark this answers is not in the document]
Comment (weijunyu) [reply apps/src/terms/TermsApp.vue:21]: Hiding for now until we decide whether we want to support this feature; currently not really used

**Review by teochengyong — APPROVED** [review]

**Pull request merged**